libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -639,6 +725,12 @@ protected virtual void OnSetScopedServices(DialogContext dialogContext)
             }
         }
 
+        /// <summary>
+        /// Removes the current most action from the given <see cref="ActionContext"/> if there are any.
+        /// </summary>
+        /// <param name="actionContext">The <see cref="ActionContext"/> for the current turn of conversation.</param>
+        /// <param name="cancellationToken">Optional, a <see cref="CancellationToken"/> that can be used by other objects.</param>
```

Comment:
How so? I understand the observation for the other cases, but not for this one.

Reply:
The complete description that's been used is:
/// <param name="cancellationToken">Optional, a <see cref="CancellationToken"/> that can be used by other objects
/// or threads to receive notice of cancellation.</param>

Otherwise, use this short one:
/// <param name="cancellationToken">Optional, a <see cref="CancellationToken"/> for the task.</param>

Reply:
Oh, now I see it, I was under the impression you were originally referring to the summary, I'll correct this right away.